### PR TITLE
Sayalis/feature 24 own recent post

### PR DESCRIPTION
**Pull request by sayalis-cuelogic**
@onkartibe @aloka-cuelogic 

Card number 24
Card link https://trello.com/c/rYnI8YQW

Completed view own recent posts.

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -12,74 +12,80 @@
...
+        <div class='clear'></div>
+        <div class='clear'></div>
+        {% block content %}{% endblock %}
+    </div>
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+    <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+    {% block scripts_extend %}{% endblock %}
+</body>
 
 </html>
```
Comment (aloka-cuelogic) [inline templates/base.html]: add new line at the EOF

File: templates/base.html [hunk 2]
```diff
@@ -12,74 +12,80 @@
...
-                            </div>
-                            <div class="col-sm-2" id="addpost">
-                                <a id="addblog" href="{% url 'post:postpage'%}">Add Post</a>
-                            </div>
-                            <div class="col-sm-2" id="addpost">
-                                <a id="update" href="{% url 'post:updatepage'%}">
+    <div id='header-area'>
+        <div id='header-place'>
+            <div id='nav-bar' class="header">                    
+                {%if request.user.id%}
```
Comment (aloka-cuelogic) [inline templates/base.html]: use request.authenticated here

File: post/views.py [hunk 3]
```diff
@@ -192,3 +192,16 @@ def get(self, request):
                 return HttpResponse(1)
         else:
             return HttpResponse(1)
+
+
+class RecentPostView(FormView):
+    template_name = 'post/blogpage.html'
+
+    def get_queryset(self):
+        return User.objects.all()
```
Comment (aloka-cuelogic) [inline post/views.py]: why this is used?
User.objects.all()

Why this is required ?

**Comment by sayalis-cuelogic** [comment]
Worked on review comment.

**Comment by aloka-cuelogic** [comment]
Reviewed

**Pull request merged**